flocker/acceptance/obsolete/test_containers.py

```diff
@@ -11,7 +11,7 @@
 from twisted.trial.unittest import TestCase
 
 from ...common import loop_until
-from ...testtools import random_name
+from ...testtools import flaky, random_name
 from ..testtools import (
     require_cluster, require_moving_backend, create_dataset,
```

Comment:
same comment as above,
having two modules with the same name is confusing, my first look didn't see the extra '.' on my laptop screen and assumed it was two different imports from the same module
I